bindings/nodejs/lib/WalletApi.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountId, CreateAccountPayload } from '../types/account'
+import { AccountManagerOptions } from '../types'
+import { AccountManager } from './AccountManager'
+
+type ProfileManager = Awaited<ReturnType<typeof AccountManager>>
+type RecoverAccounts = NonNullable<Parameters<ProfileManager['recoverAccounts']>>
+interface ProfileManagers {
+    [key: string]: ProfileManager
+}
+const profileManagers: ProfileManagers = {}
+// @ts-ignore
+export function WalletApi(): any {
```

Comment:
Why is only this subset of methods needed here?

Reply:
Those methods were in the Firefly side before on the mobile version. On the desktop version are loaded in the Electron Context Bridge here https://github.com/iotaledger/firefly/blob/develop/packages/desktop/electron/preload.js#L72

My reasoning to put them here was that they are as well wallet plugin methods that in mobile version could benefit of type definitions, but that `any` type shuold be removed for this. 
When we have changed to replace classes for factory functions, there was a Typescript error here, the `any` type definition it's just a workaround to get merged the PR faster. Added a task here https://github.com/iotaledger/wallet.rs/issues/1962